### PR TITLE
Add a use case driven guide for using encrypted expressions to replace sentitive information specified using clear-text

### PR DESCRIPTION
**Pull request by PrarthonaPaul**

### CODE REVIEW

File: _posts/2024-03-19-credential-store-for-encrypted-expressions.adoc [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+    history --enable
+```
+== Summary
+This guide demonstrates three use cases where we can use encrypted expressions to specify sensitive information. 
+
+== Resources
+* https://docs.wildfly.org/30/WildFly_Elytron_Security.html#EncryptedExpressions[Encrypted Expressions]
+* https://docs.wildfly.org/30/WildFly_Elytron_Security.html#CredentialStore[Credential Stores]
+* https://docs.wildfly.org/30/wildscribe/system-property/index.html[System Properties]
+*
```
Comment (skyllarr) [inline _posts/2024-03-19-credential-store-for-encrypted-expressions.adoc]: minor, superfluous asterisk

File: _posts/2024-03-19-credential-store-for-encrypted-expressions.adoc [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+        "populate" => true
+    }
+}
+``` 
+Here is an example of the contents of the newSecretKeyCred.cs file: 
+```
+# Properties Credential Store (Do Not Modify)
+key=RUxZAUsOhzQhCv7KbD9f2SGBgWkmTC9l0lgu28FWM4UfRBY7QQ==
+```
+As you can see there as been a key created for you already using the default alias `key`. You can create a new key with a new alias of your own using the following command:
```
Comment (skyllarr) [inline _posts/2024-03-19-credential-store-for-encrypted-expressions.adoc]: / as been/ has been

**Review by skyllarr — APPROVED** [review]
LGMT!

File: _posts/2024-03-19-credential-store-for-encrypted-expressions.adoc (hunk 1 above)
Comment (PrarthonaPaul) [reply _posts/2024-03-19-credential-store-for-encrypted-expressions.adoc]: fixed!

File: _posts/2024-03-19-credential-store-for-encrypted-expressions.adoc (hunk 2 above)
Comment (PrarthonaPaul) [reply _posts/2024-03-19-credential-store-for-encrypted-expressions.adoc]: fixed!

**Review by luck3y — APPROVED** [review]

**Comment by PrarthonaPaul** [comment]
Superseded by: https://github.com/wildfly-security/wildfly-elytron/pull/2180

**Pull request closed without merging**